buildSrc/src/main/kotlin/PropertyRetriever.kt

```diff
@@ -5,7 +5,7 @@
 
 import org.gradle.api.Project
 import java.io.File
-import java.util.Properties
+import java.util.*
```

Comment:
This doesn't look right.. Shouldn't we just import what we need instead of *?

Reply:
Fixed the regression in https://github.com/awslabs/smithy-rs/pull/1331/commits/73b34d0ec0c0c4c2aaf0e160c5fd03ab9567f404.